src/app/services/website/common/validateAndUpdate.ts

```diff
@@ -48,21 +48,43 @@ function validateSchema(data : any) {
 	if (!isObject(data)) {
 		throw new SchemaValidationError('Invalid feedback data');
 	}
-	const dataArrays: string[] = ['hosts', 'chiefEditors', 'feedbackEmailOverride'];
+	const dataArrays: string[] = ['hosts', 'chiefEditors'];
+	const overrideArrays: string[] = ['feedbackEmail', 'fallbackFeedbackEmail', 'escalationEmail'];
 
-	dataArrays.forEach((item: string) => {
-		if (item in data && (!data[item] || !Array.isArray(data[item]))) {
-			throw new SchemaValidationError(`Invalid website data: ${item} must be an array`);
-		}
-	});
+	dataArrays.forEach((item: string) =>
+		arrayValidation(
+			`Invalid website data: "${item}" must be an array`,
+			item,
+			data));
+
+	if (data.overrideSettings && data.overrideSettings.overrides) {
+		overrideArrays.forEach(item => arrayOfEmailsValidation(item, data.overrideSettings.overrides));
+	}
+}
 
-	if (data.feedbackEmailOverride) {
-		data.feedbackEmailOverride.forEach((item) => {
-			const test = emailRegexp.test(item);
-			if (!test) {
-				throw new SchemaValidationError(
-					`Invalid website data: feedbackEmailOverride must contain valid emails`);
-			}
-		});
+function arrayValidation (errText: string, item: any, data: any) {
+	if (item in data && (!data[item] || !Array.isArray(data[item]))) {
+		throw new SchemaValidationError(errText);
+	}
+}
+
+function emailValidation (errText: string, item: string) {
+	const test = emailRegexp.test(item);
+	if (!test) {
+		throw new SchemaValidationError(errText);
+	}
+}
+
+function arrayOfEmailsValidation (item: any, data: any) {
+	if (item in data) {
```

Comment:
Why don't you use function arrayValidation you wrote several lines before?

Reply:
Because of it has different principles of work and has additional check I can not add to previous one